Source/WebKit/WebProcess/GPU/media/RemoteAudioDestinationProxy.cpp

```diff
@@ -121,18 +121,19 @@ IPC::Connection* RemoteAudioDestinationProxy::connection()
         m_destinationID = RemoteAudioDestinationIdentifier::generate();
 
         m_lastFrameCount = 0;
-        SharedMemory::Handle frameCountHandle;
+        std::optional<SharedMemory::Handle> frameCountHandle;
         if ((m_frameCount = SharedMemory::allocate(sizeof(std::atomic<uint32_t>)))) {
-            if (auto localFrameHandle = m_frameCount->createHandle(SharedMemory::Protection::ReadWrite))
-                frameCountHandle = WTFMove(*localFrameHandle);
+            frameCountHandle = m_frameCount->createHandle(SharedMemory::Protection::ReadWrite);
         }
         gpuProcessConnection->connection().send(Messages::RemoteAudioDestinationManager::CreateAudioDestination(m_destinationID, m_inputDeviceId, m_numberOfInputChannels, m_outputBus->numberOfChannels(), sampleRate(), m_remoteSampleRate, m_renderSemaphore, WTFMove(frameCountHandle)), 0);
```

Comment:
If you could check if you can change the message `CreateAudioDestination` to accept only `SharedMemory::Handle` instead of `optional<...>`?
If that works, add the release assert here:
`RELEASE_ASSERT(frameCountHandle); // FIXME: the same fixme url.`